javaparser-core/src/main/javacc/java.jj

```diff
@@ -673,6 +673,63 @@ TOKEN :
  * Program structuring syntax follows.
  */
 
+ /**
+ * The stub files are represented by the {@link StubUnit}
+ */
+StubUnit StubUnit():
+{
+	/* Represents the list of compilation units that contained in a stub file */
+	NodeList<CompilationUnit> compilationUnits = emptyList();
+	CompilationUnit cu;
+
+	/*
+	* Represents the particular compilation unit components in each iteration of
+	* reading compilation units from the stub file
+	*/
+	PackageDeclaration pakage = null;
```

Comment:
There is still inconsistent use of tabs vs. spaces in this file.
(The lines that start with `*` aren't properly indented either.)